frontend/packages/dev-console/integration-tests/features/topology/topology-display_option.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Display options in topology
+	As a user, I want to see desplay options  
+
+Background:
+    Given user is in topology
+
+@regression, @smoke
+Scenario: Display options menu in topology : T-07-TC10
+   Given topology has workloads
+   When user clicks on Display Options on top of topology
+   And user sees "Pod Count", "Event Sources" if present and "Labels" under "Show" and "Expand" have options according to their presence like "Application Groupings", "Helm Releases", "Knative Services" and "Operator Groupings"
```

Comment:
Don't use if statements in when conditions, if there is any specific pre condition please mention it in given statement